<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="3" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pathsWidget">
      <attribute name="title">
       <string>Paths</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QGroupBox" name="paths">
         <property name="title">
          <string>CEN64 Files</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QGridLayout" name="pathsLayout">
            <item row="0" column="1">
             <widget class="QLineEdit" name="cen64Path"/>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="cen64Button">
              <property name="text">
               <string>Browse...</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QPushButton" name="catalogButton">
              <property name="text">
               <string>Browse...</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="pifPath"/>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="pifButton">
              <property name="text">
               <string>Browse...</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="catalogPathLabel">
              <property name="text">
               <string>ROM catalog:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="catalogPath"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="romPathLabel">
              <property name="text">
               <string>ROM directory:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QPushButton" name="romButton">
              <property name="text">
               <string>Browse...</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="cen64PathLabel">
              <property name="text">
               <string>CEN64 executable: </string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="pifPathLabel">
              <property name="text">
               <string>pifrom/pifdata file:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="romPath"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="saves">
         <property name="title">
          <string>Save Files</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <layout class="QGridLayout" name="savesLayout">
            <item row="0" column="2">
             <widget class="QPushButton" name="savesButton">
              <property name="text">
               <string>Browse...</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="savesPathLabel">
              <property name="text">
               <string>Save directory:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="savesPath"/>
            </item>
            <item row="1" column="1">
             <widget class="QCheckBox" name="saveOption">
              <property name="text">
               <string>Specify individual files for saves</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="eepromPath"/>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="sramPath"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="eepromPathLabel">
              <property name="text">
               <string>EEPROM file:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="sramPathLabel">
              <property name="text">
               <string>SRAM file:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QPushButton" name="eepromButton">
              <property name="text">
               <string>Browse...</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QPushButton" name="sramButton">
              <property name="text">
               <string>Browse...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="pathsSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="columnsWidget">
      <attribute name="title">
       <string>Columns</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="2">
        <spacer name="columnsHSpacer2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <layout class="QGridLayout" name="columnsLayout">
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <item row="1" column="0">
          <widget class="QListWidget" name="availableList">
           <property name="maximumSize">
            <size>
             <width>160</width>
             <height>250</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QWidget" name="sortWidget" native="true">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>80</height>
            </size>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="sortLayout" stretch="0,0">
              <item>
               <widget class="QToolButton" name="sortDownButton">
                <property name="text">
                 <string>...</string>
                </property>
                <property name="arrowType">
                 <enum>Qt::UpArrow</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="sortUpButton">
                <property name="text">
                 <string>...</string>
                </property>
                <property name="arrowType">
                 <enum>Qt::DownArrow</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QListWidget" name="currentList">
           <property name="maximumSize">
            <size>
             <width>160</width>
             <height>250</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="currentLabel">
           <property name="text">
            <string>Current:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="availableLabel">
           <property name="text">
            <string>Available:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QCheckBox" name="stretchOption">
           <property name="text">
            <string>Stretch first column</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QWidget" name="toggleWidget" native="true">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>80</height>
            </size>
           </property>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="toggleLayout">
              <item>
               <widget class="QToolButton" name="addButton">
                <property name="text">
                 <string>...</string>
                </property>
                <property name="arrowType">
                 <enum>Qt::RightArrow</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="removeButton">
                <property name="text">
                 <string>...</string>
                </property>
                <property name="arrowType">
                 <enum>Qt::LeftArrow</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <spacer name="columnsVSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <spacer name="columnsHSpacer1">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="otherWidget">
      <attribute name="title">
       <string>Other</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QGridLayout" name="otherLayout">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="outputLabel">
           <property name="text">
            <string>Output to Console:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="outputOption">
           <property name="toolTip">
            <string extracomment="If checked, CEN64's output will be redirected to CEN64-Qt so you can view the VI/s or debug information."/>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="otherVSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <spacer name="otherHSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttonBox</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>cen64Path</tabstop>
  <tabstop>pifPath</tabstop>
  <tabstop>romPath</tabstop>
  <tabstop>catalogPath</tabstop>
  <tabstop>cen64Button</tabstop>
  <tabstop>pifButton</tabstop>
  <tabstop>romButton</tabstop>
  <tabstop>catalogButton</tabstop>
  <tabstop>savesPath</tabstop>
  <tabstop>saveOption</tabstop>
  <tabstop>eepromPath</tabstop>
  <tabstop>sramPath</tabstop>
  <tabstop>savesButton</tabstop>
  <tabstop>eepromButton</tabstop>
  <tabstop>sramButton</tabstop>
  <tabstop>availableList</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>removeButton</tabstop>
  <tabstop>currentList</tabstop>
  <tabstop>sortDownButton</tabstop>
  <tabstop>sortUpButton</tabstop>
  <tabstop>stretchOption</tabstop>
  <tabstop>outputOption</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
